DuckDuckGo/PrivacyProtectionOverviewController.swift

```diff
@@ -178,7 +178,7 @@ class ProtectionUpgradedView: UIView {
 
 private class InteractivePopRecognizer: NSObject, UIGestureRecognizerDelegate {
     
-    var navigationController: UINavigationController
+    unowned var navigationController: UINavigationController
```

Comment:
I think we should be using `weak` here. UINavigationController life cycle is finite and in case there is an issue or async code that holds `InteractivePopRecognizer` in memory longer than NavCntrl lives, we risk a crash.